src/query/storages/system/src/columns_table.rs

```diff
@@ -298,12 +289,62 @@ pub(crate) async fn dump_tables(
                 final_dbs.push((db.to_string(), db_id));
             }
         }
+    } else {
+        let catalog_dbs = visibility_checker.get_visibility_database();
+        if let Some(catalog_dbs) = catalog_dbs {
+            for (catalog_name, dbs) in catalog_dbs {
+                if catalog_name == CATALOG_DEFAULT {
+                    let mut catalog_db_ids = vec![];
+                    for (db_name, db_id) in dbs {
+                        if let Some(db_name) = db_name {
+                            let db_id = catalog
+                                .get_database(&tenant, db_name)
```

Comment:
Now I can call get_database to get the db that user can be accessed.

But maybe we can provide a new API mget_database(db_names: &[string]) to reduce the meta call? @drmingdrmer

Reply:
I can provide db_names and I want a Vec<DatabaseInfo>

Reply:
Merge this:
- https://github.com/datafuselabs/databend/pull/16425